actions/templates/delete/index.js

```diff
@@ -64,11 +65,14 @@ async function main (params) {
       return errorResponse(401, [errorMessage(ERR_RC_INVALID_IMS_ACCESS_TOKEN, error.message)], logger);
     }
 
-    // check if the token belongs to an admin
-    const isCallerAdmin = await isAdmin(accessToken, imsUrl, adminImsOrganizations);
-    if (isCallerAdmin !== true) {
-      const err = 'This operation is available to admins only. To request template removal from Template Registry, please, create a "Template Removal Request" issue on https://github.com/adobe/aio-template-submission';
-      return errorResponse(403, [errorMessage(ERR_RC_PERMISSION_DENIED, err)], logger);
+    // check if the token is a service token and has access to the required scopes
+    if (!isValidServiceToken(accessToken, requiredScopes)) {
```

Comment:
Are we checking if the service token is valid or invalid here? lil bit confused here since it is true when the token is invalid

Reply:
Ahh gotcha